plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/wrapper/LibvirtMigrateCommandWrapper.java

```diff
@@ -175,6 +178,12 @@ Use VIR_DOMAIN_XML_SECURE (value = 1) prior to v1.0.0.
                     s_logger.debug(String.format("Replaced disk mount point [%s] with [%s] in VM [%s] XML configuration. New XML configuration is [%s].", oldIsoVolumePath, newIsoVolumePath, vmName, xmlDesc));
                 }
             }
+
+            // Replace CDROM ISO path
+            String oldCdromIsoPath = getOldVolumePathForCdrom(disks, vmName);
+            String newCdromIsoPath = getNewVolumePathForCdrom(libvirtComputingResource, conn, to);
+            xmlDesc = replaceCdromIsoPath(xmlDesc, vmName, oldCdromIsoPath, newCdromIsoPath);
```

Comment:
@weizhouapache it could be a wild idea but a simple string replace oldpath to newpath won't work?

Reply:
@shwstppr 
 my original thought is actually same as it :-D
at the end I decided to implement the same as other replacements.
it looks complex, but in my testing it took only around 0.01 second.

Reply:
Do we need any check if both old and new ISO path are same?

Reply:
@shwstppr 
yes, let me add it